constantinCurtis/src/main/java/data/collection/task/classes/CompanyOrder.java

```diff
@@ -0,0 +1,22 @@
+package data.collection.task.classes;
+
+
+import java.util.List;
+
+public class CompanyOrder extends AbstractOrder {
+    private Company company;
+
+
+    public CompanyOrder(List<Item> items, Company company) {
+        super(items);
+        this.company = company;
+    }
+
+    public Company getCompany() {
```

Comment:
you are never using this method, still we need it?

Reply:
thanks, deleted